.rhcicd/clowdapp-backend.yaml

```diff
@@ -16,6 +16,8 @@ objects:
     - notifications-engine
     - rbac
     - sources-api
```

Comment:
Sources-api is now prensent on optional dependencues, but it is still present on required dependency as well.
Is it a simple typo ?

Reply:
Oops! Nice catch :fishing_pole_and_fish: ! It should only be listed in the optional dependencies. Thank you!